matrix/buffer.py

```diff
@@ -685,10 +718,29 @@ def _membership_message(self, user, message_type):
 
         return message
 
+    def _nick_change_message(self, user, old_nick, new_nick):
+        action_color = "purple"
```

Comment:
Purple is not a valid weechat color, you can see the available colors with the `/color` command after inputing e in the newly created buffer.